app/docs/src/roadmap/index.md

```diff
@@ -59,13 +67,19 @@ Read our
 [announcement blog post](https://www.systeminit.com/blog/announcing-approval-workflows)
 to learn more about our approval workflows
 
-### Audit History  (Generally Available)
+### Audit History (Generally Available)
 
 We are providing the functionality that allows users of a workspace to see every
 change that has happened, who made them and when. The first iteration of this
 work will allow users to be able to have basic sorting and filtering of events.
 
-## Views  (Generally Available)
+### Fine Grained Access Control (In planning)
+
+We are providing the ability to attach approvers to specific sets of components,
```

Comment:
I think it's okay we're not referencing the Nouns here, as it's a less formal document

Reply:
I will be adding the corresponding blog post and opportunity readout early in the week so more will get added here